internal/action/sync.go

```diff
@@ -23,6 +25,30 @@ func (s *Action) Sync(c *cli.Context) error {
 	return s.sync(ctxutil.WithGlobalFlags(c), c.String("store"))
 }
 
+func (s *Action) autoSync(ctx context.Context) error {
+	if !ctxutil.IsInteractive(ctx) {
+		return nil
+	}
+
+	if !ctxutil.IsTerminal(ctx) {
+		return nil
+	}
+
+	if sv := os.Getenv("GOPASS_NO_AUTOSYNC"); sv != "" {
```

Comment:
We might want to document this new variable in https://github.com/gopasspw/gopass/blob/master/docs/config.md

Reply:
Done.